hacker_news_to_sqlite/cli.py

```diff
@@ -124,6 +124,15 @@ def ensure_tables(db):
             {"id": int, "type": str, "by": str, "time": int, "title": str, "text": str},
             pk="id",
         )
+    # includes hidden columns
+    all_column_names = {
+        c[1] for c in db.execute("PRAGMA table_xinfo([items])").fetchall()
```

Comment:
Generated columns are hidden and are thus not included in `db['items'].column_dict`